packages/components/src/lib/controllers/app-controller.js

```diff
@@ -1179,16 +1179,14 @@ export default class AppController {
     this._store.commit('app/lexicalRequestFinished')
   }
 
-  onMorphDataReady () {
-    this._store.commit('ui/addMessage', this.api.l10n.getMsg('TEXT_NOTICE_MORPHDATA_READY'))
```

Comment:
This message has been moved to the beginning of the `ON_HOMONYM_READY`. It is always published right after the `MORPH_DATA_READY`.